third_party/blink/renderer/modules/cobalt/h5vcc_metrics/h_5_vcc_metrics.cc

```diff
@@ -179,4 +192,9 @@ void H5vccMetrics::Trace(Visitor* visitor) const {
   EventTargetWithInlineData::Trace(visitor);
```

Comment:
I think `h5vcc_metrics_promises_` needs to be added to the
`Trace()` method, right?

Reply:
Done